src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -30,40 +27,39 @@
 	@Column(unique = true)
 	private String name;
 	private String color;
-	@OneToMany(mappedBy = "line", cascade = CascadeType.ALL)
-	private final List<Section> sections = new ArrayList<>();
+	@Embedded
+	private final Sections sections = new Sections();
```

Comment:
```suggestion
	private final Sections sections;
```

Sections.of 같은 팩토리 메서드를 추가해서 생성자에서 처리하도록 바꿔보면 어떨까요?